ledger/ledger-on-memory/BUILD.bazel

```diff
@@ -123,6 +123,7 @@ da_scala_library(
 da_scala_binary(
     name = "app",
     main_class = "com.daml.ledger.on.memory.Main",
+    max_heap_size = "2g",
```

Comment:
ledger-on-memory requires lot of memory, and prone to produce failures sporadically due to timeouts.
this is needed to prevent occasional OOMs